### PR TITLE
Update Powertoys docs

### PR DESCRIPTION
**Pull request by Jay-o-Way**
## All pages:

- https://github.com/microsoft/PowerToys/issues/12993 → text updates

## Specific pages:

- VCM → added image
- https://github.com/microsoft/PowerToys/issues/309 → added [!tip]
- https://github.com/microsoft/PowerToys/issues/9604 → added [!tip]
- https://github.com/microsoft/PowerToys/issues/1796 → added [!note]
- https://github.com/microsoft/PowerToys/issues/11363 → added [!important]
- https://github.com/microsoft/PowerToys/issues/13061 → text update (new image = #3442)
- https://github.com/microsoft/PowerToys/issues/14076#issuecomment-956034012 → added comment on VS Redistributable
- https://github.com/microsoft/PowerToys/issues/11842 → text updates (TO DO: nav menu \#help)

Awake in menu

## Possible future additions?

- https://github.com/microsoft/PowerToys/issues/12711
- https://github.com/microsoft/PowerToys/issues/8440

FYI @crutkas @dedavis6797

### CODE REVIEW

**Comment by PRMerger10** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by ktoliver** [comment]
Hi @Jay-o-Way @mattwojo - In this repository, to reference an image in an MD file, the image must be added to the repo as a file in the pull request. This is a blocking item in the internal pull request quality criteria. Thanks.

**Comment by Jay-o-Way** [comment]
> image must be added to the repo

Done.

Question though: for the issue about PowerToys Awake naming, the text in the _menu_ would like to be updated too. I don't know or see how to get there. Is that something I can do or does that need special access?

**Comment by ktoliver** [comment]
@Jay-o-Way I don't know how to request changes to the UI. @mattwojo Do you know?

**Comment by PRMerger12** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dedavis6797** [comment]
@Jay-o-Way are you suggesting to rename Awake to PowerToys Awake in the menu? If so, I can make that update on the PR I'm preparing for v0.49.

**Comment by PRMerger16** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jay-o-Way** [comment]
> @Jay-o-Way are you suggesting to rename Awake to PowerToys Awake in the menu? If so, I can make that update on the PR I'm preparing for v0.49.

For the [online docs](https://docs.microsoft.com/en-us/windows/powertoys/awake), that is - see image in first comment. I don't know how to acces it. I mean, since three pages are "below" Overview, I don't know where the hierarchy/source of the menu is. Should there be a `toc.yml` in the folder or something?

**Comment by PRMerger10** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/shortcut-guide.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ This guide uses PowerToys to display common keyboard shortcuts that use the Wind
...
+Open the shortcut guide with the shortcut key combination: <kbd>⊞ Win</kbd> + <kbd>?</kbd> _(this may require using the Shift key)_ or hold down the <kbd>⊞ Win</kbd> for the time as set in the Settings. An overlay will appear showing keyboard shortcuts that use the Windows key, including:
 
 - common Windows shortcuts
 - shortcuts for changing the position of the active window
 - taskbar shortcuts
 
 ![Screenshot of shortcut overlay](../images/pt-shortcut-guide-large.png)
 
-Keyboard shortcuts using the Windows key <kbd>⊞ Win</kbd> can be used while the guide is displayed. The result of those shortcuts (active window moved, arrow shortcut behavior changes, etc) will be displayed in the guide.
+Keyboard shortcuts using the Windows key <kbd>⊞ Win</kbd> can be used while the guide is displayed. The result of those shortcuts (active window moved, arrow shortcut behavior changes et cetera) will be displayed in the guide.
```
Comment (crutkas) [inline hub/powertoys/shortcut-guide.md]: revert this change.  , etc is standard phrase

**Review by crutkas — APPROVED** [review]
for 99%, this is fine

**Comment by PRMerger16** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger5** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jay-o-Way** [comment]
@mattwojo what's your opinion?

**Comment by mattwojo** [comment]
> > image must be added to the repo
> 
> Done.
> 
> Question though: for the issue about PowerToys Awake naming, the text in the _menu_ would like to be updated too. I don't know or see how to get there. Is that something I can do or does that need special access?

Hey @Jay-o-Way -- Trying to make sure that I understand the issue here... so the intention is for the Awake feature to be officially called "PowerToys Awake" and it needs to be renamed as such in the TOC? (The left-side table of contents menu). The TOC menu file for PowerToys is located here: https://github.com/MicrosoftDocs/windows-uwp/blob/e45f5e769fad664fb3105283494e47282acf5ab5/hub/dev-environment/toc.yml
I'm happy to make that change.. just want to be sure I'm understanding correctly.

File: hub/powertoys/image-resizer.md [hunk 2]
```diff
@@ -60,7 +60,11 @@ For example, setting the filename format to: `%1 (%2)` on the file `example.png`
 
 You can specify a directory in the filename format to group resized images into sub-directories. For example, a value of `%2\%1` would save the resized image(s) to `Small\example.jpg`
 
-You can also choose to retain the original *last modified* date on the resized image.
+<!-- [Jay] remove following warning if/when a solution is committed -->
+> [!IMPORTANT]
+> As mentioned above, you can use a backslash to create a new folder upon execution. Please be aware that, at this point - until a fix is committed, all special charaters are not filtered. For example: giving a preset a name with "1/4" in it, will also unintentionally create a folder. A colon `:` will result in corrupted files. For a complete list, see https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#file-and-directory-names.
```
Comment (mattwojo) [inline hub/powertoys/image-resizer.md]: We should make links like this one relative, so a more standard way to link this section is, that doesn't include a 'hardcoded' language-location in the URL (including the 'en-us' will force the link to show up in English, excluding it will allow the browser to choose based on the current language setting), so here's my recommended change:
 For a complete list, see [File and Directory Names](/windows/win32/fileio/naming-a-file#file-and-directory-names) in the WIN32 Local File Systems docs.

**Comment by PRMerger14** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]
GREAT updates!! Thank you Jay!!!

**Comment by PRMerger20** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jay-o-Way** [comment]
I see you solved both your comments. Thanks. That's why we have you to check 🙂 the section with the link in it will be obsolete soon, so it will probably be removed in the future anyway.
